src/confdb/data/ConfigurationModifier.java

```diff
@@ -3,976 +3,917 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 
-
 /**
- * ConfigurationModifier
- * ---------------------
+ * ConfigurationModifier ---------------------
+ * 
  * @author Philipp Schieferdecker
  *
- * retrieve only selected parts of a configuraiton,
- * after applying configurable filter decisions.
+ *         retrieve only selected parts of a configuraiton, after applying
```

Comment:
this in indentation does not seem correct

Reply:
Will correct. I guess Ecliipse is not perfect.